tests/Unit/Domain/ShoppingListTest.php

```diff
@@ -15,12 +15,13 @@ public function testConstructWithoutList(): ShoppingList
     {
         $list = new ShoppingList(
             $slug = new Slug('my-list'),
-            'My List',
+            'My List'
```

Comment:
Comment: the trailing comma should not have been removed here. It's good not to change lines that have nothing to do with your changes, plus the trailing comma here is correct.